### PR TITLE
Add 2.8 release notes

### PR DESCRIPTION
**Pull request by xluo-aws**
*Issue #:https://github.com/opensearch-project/index-management/issues/787

*Description of changes:Add 2.8 release notes

*CheckList:*
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/index-management/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Version 2.8.0.0 2023-05-26
+
+Compatible with OpenSearch 2.8.0
+
+### Maintenance
+* Fix build errors after refactoring of Strings /*
```
Comment (SuZhou-Joe) [inline release-notes/opensearch-index-management.release-notes-2.8.0.0.md:6]: The license header should not be here I think?

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```
Comment (Hailong-am) [inline release-notes/opensearch-index-management.release-notes-2.8.0.0.md:9]: copyright part should not needed

**Review by SuZhou-Joe — DISMISSED** [review]

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+## Version 2.8.0.0 2023-05-26
```
Comment (gaobinlong) [inline release-notes/opensearch-index-management.release-notes-2.8.0.0.md:1]: 2.8.0 will be released on June 6th, so change the date to that day?

Comment (xluo-aws) [reply release-notes/opensearch-index-management.release-notes-2.8.0.0.md]: Done

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md (hunk 1 above)
Comment (xluo-aws) [reply release-notes/opensearch-index-management.release-notes-2.8.0.0.md]: Removed, it's copied by mistake

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md (hunk 2 above)
Comment (xluo-aws) [reply release-notes/opensearch-index-management.release-notes-2.8.0.0.md]: Removed, copied by mistake

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+### Maintenance
+* Fix build errors after refactoring of Strings class in opensearch core. ([#786](https://github.com/opensearch-project/index-management/pull/786))
+* Upgrade to gradle 8.1.1. ([#777](https://github.com/opensearch-project/index-management/pull/777))
+* Bump version to 2.8. ([#787](https://github.com/opensearch-project/index-management/pull/787))
+
+### Features
+* Support notification integration with long running operations. ([#790, 791, 793](https://github.com/opensearch-project/index-management/pull/793))
+
+### Refactoring
+* Replace Set in org.opensearch.common.collect with java.util references. ([#785](https://github.com/opensearch-project/index-management/pull/785))
```
Comment (Hailong-am) [inline release-notes/opensearch-index-management.release-notes-2.8.0.0.md]: #785 was in bug fixes section, duplicate here

Comment (xluo-aws) [reply release-notes/opensearch-index-management.release-notes-2.8.0.0.md]: Removed

**Review by gaobinlong — DISMISSED** [review]

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+## Version 2.8.0.0 2023-06-06
+
+Compatible with OpenSearch 2.8.0
+
+### Maintenance
+* Fix build errors after refactoring of Strings class in opensearch core. ([#786](https://github.com/opensearch-project/index-management/pull/786))
```
Comment (Hailong-am) [inline release-notes/opensearch-index-management.release-notes-2.8.0.0.md:6]: this one was not in 2.8 release

Comment (xluo-aws) [reply release-notes/opensearch-index-management.release-notes-2.8.0.0.md:6]: Is this main only?

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+## Version 2.8.0.0 2023-06-06
+
+Compatible with OpenSearch 2.8.0
+
+### Maintenance
+* Upgrade to gradle 8.1.1. ([#777](https://github.com/opensearch-project/index-management/pull/777))
+* Bump version to 2.8. ([#787](https://github.com/opensearch-project/index-management/pull/787))
```
Comment (Hailong-am) [inline release-notes/opensearch-index-management.release-notes-2.8.0.0.md:7]: should be #759, there is not PR #787, it is a issue.

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+### Features
+* Support notification integration with long running operations. ([#790, 791, 793](https://github.com/opensearch-project/index-management/pull/793))
```
Comment (Hailong-am) [inline release-notes/opensearch-index-management.release-notes-2.8.0.0.md:10]: #790 was duplicate with 791.

File: release-notes/opensearch-index-management.release-notes-2.8.0.0.md (hunk 5 above)
Comment (Hailong-am) [reply release-notes/opensearch-index-management.release-notes-2.8.0.0.md]: Yes, we didn't backport to 2.x

**Review by Hailong-am — APPROVED** [review]

**Review by gaobinlong — APPROVED** [review]

**Pull request merged**